Sources/Controllers/ALKConversationViewController+AutoComplete.swift

```diff
@@ -21,7 +25,7 @@ extension ALKConversationViewController: AutoCompletionDelegate {
                     let items = suggestionDict
                     for dictionary in items {
                         if let key = dictionary["searchKey"] as? String, let content = dictionary["message"] as? String {
-                            let autoCompleteItem = AutoCompleteItem(key: key, content: content)
```

Comment:
instead of passing nil in every place, can we make it default parameter with nil & whenever needed, we can pass the value ?